examples/next-js/landing-page/pages/[[...path]].tsx

```diff
@@ -0,0 +1,95 @@
+import type {
+  GetStaticPropsContext,
+  InferGetStaticPropsType,
+} from 'next'
+import { NextSeo } from 'next-seo'
+import { useRouter } from 'next/router'
+import { BuilderComponent, Builder, builder } from '@builder.io/react'
+import builderConfig from '@config/builder'
+import DefaultErrorPage from 'next/error'
+import Head from 'next/head'
+import { resolveBuilderContent } from '@lib/resolve-builder-content'
+import { Link } from '@components/Link/Link'
+
+
+export async function getStaticProps({
+  params,
+}: GetStaticPropsContext<{ path: string[] }>) {
+  const page = await resolveBuilderContent('page', {
+    urlPath: '/' + (params?.path?.join('/') || ''),
+  })
+
+  return {
+    props: {
+      page,
+    },
```

Comment:
```suggestion
    },
    notFound: !page,
```

it's not very well documented but it's supported for "fallback: true" a a top level option